web/styles/compose.css

```diff
@@ -797,6 +797,7 @@ a.compose_control_button.hide {
 
     &.typeahead {
         background: hsl(0, 0%, 100%);
+        right: 0;
```

Comment:
What does this change do? It seems like it applies to every typeahead, not just the search one.

Reply:
I have no idea why I put it in this selector, thanks for catching it!

Reply:
It helps the search typeahead be full width instead of this:

<img width="826" alt="image" src="https://user-images.githubusercontent.com/5634097/227745122-16d75045-62d5-4eb1-8c1e-ae60f31802c2.png">

To be aligned correctly, we made all the other changes to get it in the right parent container.

Reply:
I moved it to search-specific CSS.